src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -124,11 +159,69 @@ const AppWrapper = (props: { children: ReactNode }) => {
     }
   };
 
+  const loadDatabase = async () => {
+    const connectionsDetails = await Agent.agent.connections.getConnections();
+
+    const credentials = await Agent.agent.credentials.getCredentials();
+    const storedIdentifiers = await Agent.agent.identifiers.getIdentifiers();
+
+    dispatch(setIdentifiersCache(storedIdentifiers));
+    dispatch(setCredsCache(credentials));
+    dispatch(setConnectionsCache(connectionsDetails));
+  };
+
+  const loadPreferences = async () => {
+    const getPreferenceSafe = async (key: string) => {
+      try {
+        return await PreferencesStorage.get(key);
+      } catch (e) {
+        // TODO: handle error
```

Comment:
We were handling this before where we throw it if it's not in the format `${PreferencesStorage.KEY_NOT_FOUND} ${PreferencesKeys.APP_IDENTIFIERS_FAVOURITES}` (or whatever the key is). That shouldn't have been removed

Reply:
i just rollback this changes but keeping them inside `loadPreferences` method